### PR TITLE
Add in feature flag check that works with no entries for business typ…

### PR DESCRIPTION
**Pull request by seeker25**
…e changes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/YourCompany/ChangeBusinessType.vue [hunk 1]
```diff
@@ -376,6 +376,10 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
     if (this.getEntitySnapshot?.businessInfo?.legalType === CorpTypeCd.BC_CCC) {
       return false
     }
+    // If our feature flag is off, don't allow editing of the business type.
+    if (this.supportedEntityTypes == null || this.supportedEntityTypes?.length === 0) {
+      return false
+    }
```
Comment (severinbeauvais) [inline src/components/common/YourCompany/ChangeBusinessType.vue:382]: Should look at new FF instead.

Comment (severinbeauvais) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: I see what you're doing here... sec

Comment (severinbeauvais) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: You only load the flag when the component is mounted. Normally we'd check the flag in real-time (same call).

Comment (seeker25) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: can change it later in main

Comment (severinbeauvais) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: This flag isn't on or off. It's a list of entity types that support changing types.

So this logic isn't quite right. You need to check the current entity type.

Comment (seeker25) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: If it's empty, it wont show, because there are no supported types.

Comment (seeker25) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: It's inclusive:

image

If that's empty, it shouldn't allow any of the types

Comment (seeker25) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: We don't even need this PR - I think we can just turn off the feature flag

Comment (severinbeauvais) [reply src/components/common/YourCompany/ChangeBusinessType.vue:382]: I think you're right. We should clean up this code though, so we can allow some entity type edits.

**Pull request closed without merging**